src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewerPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
+
+interface GeneralCellStyle {
+    borderStyle: "solid" | "dashed" | "dotted" | undefined;
+    borderWidth: number;
+    borderLeftWidth: number;
+    borderTopWidth: number;
+    padding: string;
```

Comment:
Probably best to just have these as optional entries, so that for example borderLeftWidth doesn't have to be set to 0

Reply:
I removed it due to the changes of combining classes